docs/advanced/server-side-rendering.md

```diff
@@ -177,254 +177,156 @@ we'll have to import from. `preact-ssr-prepass`.
 ## Next.js
 
 If you're using [Next.js](https://nextjs.org/) you can save yourself a lot of work by using
-`next-urql`. The `next-urql` package includes setup for `react-ssr-prepass` already, which automates
-a lot of the complexity of setting up server-side rendering with `urql`.
+`@urql/next`. The `@urql/next` package is set to work with Next 13.
 
-We have a custom integration with [`Next.js`](https://nextjs.org/), being [`next-urql`](https://github.com/urql-graphql/urql/tree/main/packages/next-urql)
-this integration contains convenience methods specifically for `Next.js`.
-These will simplify the above setup for SSR.
-
-To set up `next-urql`, first we'll install `next-urql` with `react-is` and `urql` as
+To set up `@urql/next`, first we'll install `@urql/next` and `urql` as
 peer dependencies:
 
 ```sh
-yarn add next-urql react-is urql graphql
+yarn add @urql/next urql graphql
 # or
-npm install --save next-urql react-is urql graphql
+npm install --save @urql/next urql graphql
 ```
 
-The peer dependency on `react-is` is inherited from `react-ssr-prepass` requiring it.
-
-Note that if you are using Next before v9.4 you'll need to polyfill fetch, this can be
-done through [`isomorphic-unfetch`](https://www.npmjs.com/package/isomorphic-unfetch).
+We now have two ways to leverage `@urql/next`, one being part of a Server component
+or being part of the general `app/` folder.
 
-We're now able to wrap any page or `_app.js` using the `withUrqlClient` higher-order component. If
-we wrap `_app.js` we won't have to wrap any individual page.
+In a server component we will import from `@urql/next/rsc`
 
-```js
-// pages/index.js
+```ts
```

Comment:
the syntax highlighting is a bit off, maybe `tsx` would work better
![image](https://github.com/urql-graphql/urql/assets/1933330/1ed52cde-30cf-4657-800a-fe90c94f2cea)